appveyor.yml

```diff
@@ -308,6 +308,11 @@ for:
           if ($env:TARGET_ARCH -eq 'ia32') {
             $env:npm_config_arch = "ia32"
           }
+      - node script/yarn add --dev dd-trace
```

Comment:
We shouldn't be adding dependencies during CI, it is unsafe as it bypasses the lockfile